### DISCUSSION TITLE
How to rotate a component? I can’t seem to find any methods for rotating components.

### DISCUSSION THREAD

**Discussion by c61292558** [fltk-rs/fltk-rs#1436] [Q&A] [answered]
### How to rotate a component? I can’t seem to find any methods for rotating components.

**Comment by MoAlyousef** [comment, marked as answer]
There is a draw::rotate  and draw::draw_text_angled functions which can be used in a draw routine. However it has no effect on widgets. I don’t think FLTK provides a way to rodate a widget.

**Reply by c61292558** [reply]
oh,I see. Thanks for your help.
